.changeset/thin-cooks-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+"eslint-config-godaddy": minor
```

Comment:
Is there a new feature here, or should this be a patch?

Reply:
yeah thought with the amount of changes a minor made sense, but looking back it's really just a fix that was missed originally